sql/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -104,27 +99,30 @@ protected void doStart() {
         if (!enabled) {
             return;
         }
+        bootstrap = new ServerBootstrap(new NioServerSocketChannelFactory(
+            Executors.newCachedThreadPool(daemonThreadFactory(settings, "postgres-netty-boss")),
+            Executors.newCachedThreadPool(daemonThreadFactory(settings, "postgres-netty-worker"))
+        ));
+        boolean success = false;
+        try {
+            bootstrap.setOption("child.tcpNoDelay", settings.getAsBoolean("tcp_no_delay", true));
+            bootstrap.setOption("child.keepAlive", settings.getAsBoolean("tcp_keep_alive", true));
```

Comment:
afaik there is no reason to keep the setOption() calls inside the try block, I'd prefer only needed stuff there